docs/api/access-control.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: 'access-control'
+---
+
+## Overview
+
+Access Control in PMM can be used to restrict access to individual metrics.  
+Access Control is in tech preview and needs to be enabled manually from the PMM settings before it can be used.
+
+Once enabled, restricting access to metrics can be performed by:
+
+1. Creating a Percona role
+2. Assigning the role to a user
+
+### Create a Percona role
+
+```bash
+curl -X POST "http://localhost/v1/management/Role/Create" \ 
+     -H "accept: application/json" \
```

Comment:
I believe this one is not necessary and it should work without it. Just looking for cleaner code :)